packages/SwingSet/test/bundling/test-bundles.js

```diff
@@ -14,6 +16,10 @@ function bfile(name) {
   return new URL(name, import.meta.url).pathname;
 }
 
+function tmpfile(name) {
```

Comment:
Writing to a shared tmpdir enables a symlink-based confused-deputy attack (not like we're using shared machines much, but still best to avoid). Could you take a look at my abandoned PR #5180 and maybe use that approach? Keep the file in the tree, but use `t.teardown` to delete it afterwards, and also name it something not lintable like `bundle-disk.bundle`.